docs/ABOUT.md

```diff
@@ -0,0 +1,19 @@
+[*Elixir*](http://elixir-lang.org/), initially released in 2012, extends upon the already robust features of Erlang while also being easier for beginners to access, read, test, and write. Everything in Elixir is an expression.
+
+Elixir actually compiles down to [bytecode](https://en.wikipedia.org/wiki/Bytecode) and then run on the [BEAM Erlang Virtual Machine](https://erlangcentral.org/euc-2014-robert-virding-hitchhikers-tour-of-the-beam/#.VmsIIBorJE4).
+
+The asynchronous communication implementation allows the code to be lightweight, yet incorporate high-volume concurrency.
+
+José Valim, the creator of Elixir, explains [here](https://vimeo.com/53221562) how he built the language for applications to be:
+1) Distributed
+2) Fault-Tolerant
+3) Soft-Real-Time
+4) Hot-Code-Swapped (can introduce new code without stopping the sever)
```

Comment:
This list still does not display as a list. Please check the "view" button here on GitHub.